src/components/dev-hub/searchbar/SearchResults.js

```diff
@@ -9,8 +9,8 @@ import SearchLoader from './SearchLoader';
 import { P } from '~components/dev-hub/text';
 
 const SEARCHBAR_HEIGHT = '36px';
-const SEARCH_RESULT_HEIGHT = '112px';
-const SEARCH_RESULT_MOBILE_HEIGHT = '136px';
+const SEARCH_RESULT_HEIGHT = '148px';
+const SEARCH_RESULT_MOBILE_HEIGHT = '172px';
```

Comment:
increasing the search result height to accommodate for result type badge.

Reply:
Excellent, this piece looks really nice!